client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -276,7 +276,7 @@ function getFallbackDestination( {
 		return siteWithReceiptOrCartUrl;
 	}
 
-	if ( siteSlug ) {
+	if ( siteSlug && ! isCartEmpty ) {
```

Comment:
Thank you! Could you add a test to cover this change in `client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js`?

Reply:
Test case added, thanks.